lib/Controller/PreferencesController.php

```diff
@@ -31,20 +31,26 @@
 use OCP\AppFramework\Controller;
 use OCP\AppFramework\Http\JSONResponse;
 use OCP\IRequest;
+use OCP\PreConditionNotMetException;
 
 class PreferencesController extends Controller {
 
 	/** @var IUserPreferences */
 	private $userPreference;
 
+	/** @var string */
+	private $UserId;
```

Comment:
```suggestion
	private $userId;
```

capitalization on the first letter is only required for the argument on the constructor. The internal member we can case correctly :)

Reply:
Ah good to know! Will change :)